Reorg astro runtime (#2726)
astronomer/docs: astro/runtime-version-lifecycle-policy.md

---

> title: "Astro Runtime maintenance and security"

Reviewer: I know there's a tab called "security", but that doesn't really cover what's in this doc. The "security" topic is about how Astronomer reacts to security vulnerabilities _with respect to our maintenance policy_. It's not about Runtime's security itself, which is what this title suggests.

  <meta name="og:description" content="Learn how Astronomer releases and maintains Astro Runtime. Astro Runtime is a Docker image built by Astronomer that provides a differentiated Apache Airflow experience and execution framework." />
</head>

Astro Runtime is required by all Astronomer products and is maintained by Astronomer. It improves your Airflow environment's functionality, reliability, efficiency, and performance.

> Astronomer maintenance and lifecycle policies are part of the Astro Runtime distribution and define the period that specific versions are supported and how frequently updates are provided.

Reviewer 2: ```suggestion
The Astro Runtime distribution includes the Astronomer maintenance and lifecycle policies, which defines the period that specific versions are supported and how frequently updates are provided.
```

  <meta name="og:description" content="Learn how Astronomer releases and maintains Astro Runtime. Astro Runtime is a Docker image built by Astronomer that provides a differentiated Apache Airflow experience and execution framework." />
</head>

Astro Runtime is required by all Astronomer products and is maintained by Astronomer. It improves your Airflow environment's functionality, reliability, efficiency, and performance.

Astronomer maintenance and lifecycle policies are part of the Astro Runtime distribution and define the period that specific versions are supported and how frequently updates are provided.

> ## Versioning

Reviewer: This isn't part of our policy, this is just how we release Runtime regardless of how long we support it. Feels out of place in this doc.

Author: well, lifecycle exists because of releases!
